[media-controls] add a plain variant of the overflow icon

graouts opened the pull request:
#### f3f693605e2361d756076e47b231222c5223705b
<pre>
[media-controls] add a plain variant of the overflow icon
<a href="https://bugs.webkit.org/show_bug.cgi?id=241965">https://bugs.webkit.org/show_bug.cgi?id=241965</a>

Reviewed by Dean Jackson.

Add a new plain version of the overflow icon which is just the ellipsis without a circle around it.

* Source/WebCore/Modules/modern-media-controls/controls/icon-service.js:
* Source/WebCore/Modules/modern-media-controls/images/iOS/Ellipsis.svg: Added.
* Source/WebCore/Modules/modern-media-controls/images/macOS/Ellipsis.svg: Added.

Canonical link: <a href="https://commits.webkit.org/251864@main">https://commits.webkit.org/251864@main</a>
</pre>

dcrousso commented on Source/WebCore/Modules/modern-media-controls/images/iOS/OverflowPlain.svg:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="92px" height="92px" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
This seems way larger than it needs to be 😅

Also, where did you get this icon?  I normally export it from SF Symbols and it usually comes a single combined `<path>` (see [this](https://github.com/WebKit/WebKit/commit/4139966c01bf5d7849026e8f664cd786b35dbe9c#diff-e8dabb94a8e8e8d8b94bb6ca08e73be1d920410271cecd7383d48e87276267ce) as an example).

nt1m replied:
```suggestion
<svg width="92" height="92" xmlns="http://www.w3.org/2000/svg">
```

dcrousso commented on Source/WebCore/Modules/modern-media-controls/controls/icon-service.js:
```diff
@@ -35,6 +35,7 @@ const Icons = {
     InvalidCircle   : { name: "InvalidCircle", type: "pdf", label: UIString("Invalid") },
     InvalidPlacard  : { name: "invalid-placard", type: "png", label: UIString("Invalid") },
     Overflow        : { name: "Overflow", type: "svg", label: UIString("More\u2026") },
+    OverflowPlain   : { name: "OverflowPlain", type: "svg", label: UIString("More\u2026") },
```
NIT: Maybe instead of calling this "plain" we call it `OverflowDots` since that's really what it's about (and also on macOS the overflow icon is ">>", so it's a bit odd to jump from ">>" to "..." by calling it "plain" 😅).

graouts replied:
I think we should just call it "ellipsis" since that's what it is.

nt1m commented on Source/WebCore/Modules/modern-media-controls/images/iOS/OverflowPlain.svg:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
This XML declaration isn't needed

nt1m commented on Source/WebCore/Modules/modern-media-controls/images/iOS/OverflowPlain.svg:
```diff
@@ -0,0 +1,8 @@
+   <path d="M44.043-27.1973Q46.2891-27.1973 48.1201-28.2715Q49.9512-29.3457 51.0254-31.1523Q52.0996-32.959 52.0996-35.2051Q52.0996-38.5742 49.7559-40.918Q47.4121-43.2617 44.043-43.2617Q40.7227-43.2617 38.3789-40.918Q36.0352-38.5742 36.0352-35.2051Q36.0352-32.959 37.1094-31.1523Q38.1836-29.3457 40.0146-28.2715Q41.8457-27.1973 44.043-27.1973Z"/>
+   <path d="M73.2422-27.1973Q75.4395-27.1973 77.2705-28.2715Q79.1016-29.3457 80.2002-31.1523Q81.2988-32.959 81.2988-35.2051Q81.2988-38.5742 78.9307-40.918Q76.5625-43.2617 73.2422-43.2617Q69.8242-43.2617 67.4316-40.918Q65.0391-38.5742 65.0391-35.2051Q65.0391-32.959 66.1377-31.1523Q67.2363-29.3457 69.0918-28.2715Q70.9473-27.1973 73.2422-27.1973Z"/>
+  </g>
+</svg>
```
```suggestion
  <path d="M14.8926 52.8027q2.2949 0 4.1504-1.0742 1.8554-1.0742 2.9541-2.8808 1.0986-1.8067 1.0986-4.0528 0-3.3691-2.3926-5.7129-2.3926-2.3437-5.8105-2.3437-3.3203 0-5.6885 2.3437-2.36816 2.3438-2.36816 5.7129 0 2.2461 1.09863 4.0528 1.09863 1.8066 2.92973 2.8808 1.831 1.0742 4.0283 1.0742ZM44.043 52.8027q2.2461 0 4.0771-1.0742 1.8311-1.0742 2.9053-2.8808 1.0742-1.8067 1.0742-4.0528 0-3.3691-2.3437-5.7129-2.3438-2.3437-5.7129-2.3437-3.3203 0-5.6641 2.3437-2.3437 2.3438-2.3437 5.7129 0 2.2461 1.0742 4.0528 1.0742 1.8066 2.9052 2.8808 1.8311 1.0742 4.0284 1.0742ZM73.2422 52.8027q2.1973 0 4.0283-1.0742 1.8311-1.0742 2.9297-2.8808 1.0986-1.8067 1.0986-4.0528 0-3.3691-2.3681-5.7129-2.3682-2.3437-5.6885-2.3437-3.418 0-5.8106 2.3437-2.3925 2.3438-2.3925 5.7129 0 2.2461 1.0986 4.0528 1.0986 1.8066 2.9541 2.8808 1.8555 1.0742 4.1504 1.0742Z"/>
</svg>
```

This is equivalent (I used https://jakearchibald.github.io/svgomg/)

nt1m commented on Source/WebCore/Modules/modern-media-controls/images/macOS/OverflowPlain.svg:
```diff
@@ -0,0 +1,8 @@
+   <path d="M44.043-27.1973Q46.2891-27.1973 48.1201-28.2715Q49.9512-29.3457 51.0254-31.1523Q52.0996-32.959 52.0996-35.2051Q52.0996-38.5742 49.7559-40.918Q47.4121-43.2617 44.043-43.2617Q40.7227-43.2617 38.3789-40.918Q36.0352-38.5742 36.0352-35.2051Q36.0352-32.959 37.1094-31.1523Q38.1836-29.3457 40.0146-28.2715Q41.8457-27.1973 44.043-27.1973Z"/>
+   <path d="M73.2422-27.1973Q75.4395-27.1973 77.2705-28.2715Q79.1016-29.3457 80.2002-31.1523Q81.2988-32.959 81.2988-35.2051Q81.2988-38.5742 78.9307-40.918Q76.5625-43.2617 73.2422-43.2617Q69.8242-43.2617 67.4316-40.918Q65.0391-38.5742 65.0391-35.2051Q65.0391-32.959 66.1377-31.1523Q67.2363-29.3457 69.0918-28.2715Q70.9473-27.1973 73.2422-27.1973Z"/>
+  </g>
+</svg>
```
```suggestion
<svg width="18" height="18" viewBox="0 0 92 92" xmlns="http://www.w3.org/2000/svg">
  <path d="M14.8926 52.8027q2.2949 0 4.1504-1.0742 1.8554-1.0742 2.9541-2.8808 1.0986-1.8067 1.0986-4.0528 0-3.3691-2.3926-5.7129-2.3926-2.3437-5.8105-2.3437-3.3203 0-5.6885 2.3437-2.36816 2.3438-2.36816 5.7129 0 2.2461 1.09863 4.0528 1.09863 1.8066 2.92973 2.8808 1.831 1.0742 4.0283 1.0742ZM44.043 52.8027q2.2461 0 4.0771-1.0742 1.8311-1.0742 2.9053-2.8808 1.0742-1.8067 1.0742-4.0528 0-3.3691-2.3437-5.7129-2.3438-2.3437-5.7129-2.3437-3.3203 0-5.6641 2.3437-2.3437 2.3438-2.3437 5.7129 0 2.2461 1.0742 4.0528 1.0742 1.8066 2.9052 2.8808 1.8311 1.0742 4.0284 1.0742ZM73.2422 52.8027q2.1973 0 4.0283-1.0742 1.8311-1.0742 2.9297-2.8808 1.0986-1.8067 1.0986-4.0528 0-3.3691-2.3681-5.7129-2.3682-2.3437-5.6885-2.3437-3.418 0-5.8106 2.3437-2.3925 2.3438-2.3925 5.7129 0 2.2461 1.0986 4.0528 1.0986 1.8066 2.9541 2.8808 1.8555 1.0742 4.1504 1.0742Z"/>
</svg>
```

ditto

graouts committed:
[media-controls] add a plain variant of the overflow icon
https://bugs.webkit.org/show_bug.cgi?id=241965

Reviewed by Dean Jackson.

Add a new plain version of the overflow icon which is just the ellipsis without a circle around it.

* Source/WebCore/Modules/modern-media-controls/controls/icon-service.js:
* Source/WebCore/Modules/modern-media-controls/images/iOS/Ellipsis.svg: Added.
* Source/WebCore/Modules/modern-media-controls/images/macOS/Ellipsis.svg: Added.

Canonical link: https://commits.webkit.org/251864@main
